web-ui/src/main/resources/catalog/components/metadataactions/RelatedResourcesService.js

```diff
@@ -142,13 +142,16 @@
 
           var openLink = function(record, link) {
             var url = $filter('gnLocalized')(record.url) || record.url;
-            if (url && (url.indexOf('\\') == 0 ||
```

Comment:
This is used by some users to link to local network resources (CIFS link I think). Maybe this if can be easier with ```url && url.match('http|ftp|//|\\\\') != null```